heroic-component/src/main/java/com/spotify/heroic/tracing/RandomParentSampler.kt

```diff
@@ -0,0 +1,44 @@
+package com.spotify.heroic.tracing
```

Comment:
Missing license header - I'm not sure if the license check runs for kotlin files.